docs/reference/indices/get-component-template.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[getting-component-templates]] 
+=== Get component template API
+++++
+<titleabbrev>Get component template</titleabbrev>
+++++
+
+Returns information about one or more component templates.
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT /_component_template/template_1 
+{
+  "template": {
+    "settings": {
+      "index.number_of_replicas": 0
+    },
+    "mappings": {
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE /_component_template/template_*
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+GET /_component_template/template_1
+--------------------------------------------------
+
+[[get-component-template-api-request]]
+==== {api-request-title}
+
+`GET /_component-template/<component-template>`
+
+
+[[get-component-template-api-path-params]]
+==== {api-path-parms-title}
+
+`<component-template>`
+(Required, string)
+Comma-separated list of component template names used to limit the request.
+Wildcard (`*`) expressions are supported.
```

Comment:
Based on the example below, this is optional. May also want to mention that you can omit this parameter to retrieve all component templates.
```suggestion
`<component-template>`
(Optional, string)
Comma-separated list of component template names used to limit the request.
Wildcard (`*`) expressions are supported.
```